### PR TITLE
Recursion GCD 

### LINKED ISSUE #13: Recursion: GCD
Write a function that calculates the GCD (Greatest Common Divisor) of two numbers **recursively**.

Examples
```
gcd(10, 20) ➞ 10

gcd(1, 3) ➞ 1

gcd(5, 7) ➞ 1

gcd(2, 6) ➞ 2 
```

Notes
N/A

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: RecursionGCD.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+gcd(5, 7) ➞ 1
+
+gcd(2, 6) ➞ 2 
+
+Notes
+N/A
+=end
+
+def gcd(a, b)
```
Comment (danielpaul) [inline RecursionGCD.rb]: remove the printing out comments and return the answer

**Review by danielpaul — CHANGES_REQUESTED** [review]

**Comment by danielpaul** [comment]
I think you are using Git wrongly... there are all sorts of files being checked into different branches. You are committing the same files in lots of different branches. Thats not how it is supposed to work. Each branch is a different version of the codebase.

**Comment by danielpaul** [comment]
Thats also why you are having a lot of merge conflicts. Can you fix this PR so it only have the changes that it should?

**Comment by Anishluke92** [comment]
I dont how other files are getting merged into this files ! when i'm working on a branch and its file is open and further when i switch it to  another branch and work on it. when i commit on the current the pervious branch  file which is opened will also get commit to the current branch

**Comment by Anishluke92** [comment]
I think thats how merge conflict happens

**Comment by Anishluke92** [comment]
> Thats also why you are having a lot of merge conflicts. Can you fix this PR so it only have the changes that it should?

Can you give me an hint how to fix it and i head on from there

**Comment by Anishluke92** [comment]
I have fixed the merge conflict ! but i have few doubts

**Review by danielpaul — APPROVED** [review]

**Pull request merged**